src/token/common/erc2981/interface.cairo

```diff
@@ -0,0 +1,8 @@
+use starknet::ContractAddress;
+
+pub const IERC2981_ID: felt252 = 0x2d3414e45a8700c29f119a54b9f11dca0e29e06ddcb214018fc37340e165ed6;
```

Comment:
We should also consider adding the interface ID that is specified in the [EIP-2981](https://eips.ethereum.org/EIPS/eip-2981) and in the Solidity implementation (`0x2a55205a`)

Reply:
Sorry I don't understand what you mean with Solidity implementation interface ID:
- Do you mean I need to add camelCase method `royaltyInfo` ?
- If yes, should IERC2981_ID be computed with this interface added or should I add additional interface ID ?

Reply:
We have discussed the issue internally and decided that we're not going to add the support for the interface IDs from Solidity